prime-angular-frontend/src/app/modules/adjudication/shared/components/ha-authorized-user-entry/ha-authorized-user-entry.component.ts

```diff
@@ -0,0 +1,74 @@
+import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+
+import { Subscription } from 'rxjs';
+
+import { RouteUtils } from '@lib/utils/route-utils.class';
+import { HealthAuthorityResourceService } from '@core/resources/health-authority-resource.service';
+import { FormUtilsService } from '@core/services/form-utils.service';
+import { HAAuthorizedUser } from '@shared/models/ha-authorized-user.model';
+import { Role } from '@auth/shared/enum/role.enum';
+
+import { HaAuthorizedUserEntryFormState } from './ha-authorized-user-entry-form-state.class';
+
+@Component({
+  selector: 'app-ha-authorized-user-entry',
+  templateUrl: './ha-authorized-user-entry.component.html',
+  styleUrls: ['./ha-authorized-user-entry.component.scss']
+})
+export class HaAuthorizedUserEntryComponent implements OnInit {
+  public formState: HaAuthorizedUserEntryFormState;
+  public routeUtils: RouteUtils;
+  public busy: Subscription;
+  public haid: number;
+  public auid: number;
+
+  public Role = Role;
+
+  constructor(
+    protected fb: FormBuilder,
```

Comment:
Should be `private` only need to use `protected` for DI when passing to a base class through `super(...)`